### DISCUSSION TITLE
Shift+Insert is bound to Paste, but where and how to rebind it?

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#6348] [Q&A]
I find Shift+Insert is bound to Paste.

But as I cannot find where this binding has taken place,
nor what the official name of Paste is,
I cannot rebind it.

https://unix.stackexchange.com/questions/788784/in-i3-how-to-bind-controlshiftv-to-also-do-paste/

**Comment by yusufaktepe** [comment]
<kbd>Shift</kbd>+<kbd>Insert</kbd> is a special binding that works in most X11 applications. It pastes the 'PRIMARY' selection clipboard.

This has nothing to do with i3 itself. You need to install an additional clipboard manager to bind a key to something similar.

**Reply by orestisfl** [reply]
In i3
```
bindsym Shift+Insert nop
```
should do the trick
